pkg/util/util.go

```diff
@@ -207,7 +199,7 @@ func FileExists(path string) (bool, error) {
 }
 
 func EnsureDirExists(dir string) error {
-	err := os.MkdirAll(dir, 0755)
+	err := os.MkdirAll(dir, 0o755)
```

Comment:
why 0o755??

Reply:
output of `make fmt`

Reply:
Really? 755 is a permission but 0o755, I dont know what that `0o` brings

Reply:
Indeed this is an actual thing, it indicates an octal number so it's safe.